src/app/pcasts/models/user.py

```diff
@@ -17,17 +18,22 @@ class User(Base):
 
   def __init__(self, **kwargs):
     self.google_id = kwargs.get('google_id')
+    self.facebook_id = kwargs.get('facebook_id')
     self.email = kwargs.get('email', '')
     self.first_name = kwargs.get('first_name')
     self.last_name = kwargs.get('last_name')
     self.image_url = kwargs.get('image_url')
     self.followers_count = kwargs.get('followers_count', 0)
     self.followings_count = kwargs.get('followings_count', 0)
-    self.username = kwargs.get('username', 'temp-{}'.format(self.google_id))
+    non_empty_id, id_source = (self.facebook_id, "fb") if self.google_id \
+        is None else (self.google_id, "google")
```

Comment:
I'm a fan of moving as many string constants as possible to the constants.py file, even if this is only used here, it's too hard to foresee what we will need in the future and it can get messy.